pkg/reconcilermanager/controllers/util.go

```diff
@@ -206,8 +208,25 @@ const (
 )
 
 // helmSyncEnvs returns the environment variables for the oci-sync container.
-func helmSyncEnvs(repo, chart, version, releaseName, namespace string, auth configsync.AuthType, period float64) []corev1.EnvVar {
+func helmSyncEnvs(repo, chart, version, releaseName, namespace string, values *unstructured.Unstructured, valuesFiles []string, includeCRDs bool, auth configsync.AuthType, period float64) []corev1.EnvVar {
```

Comment:
There are too many parameters in this function. Consider creating a struct for it (e.g. `helmOptions`). Can be done in a separate commit or in a follow-up PR.

Reply:
I think we can, I am following the style of other env functions, I was wondering why the other env functions do not just pass in the spec directly.

Reply:
Done!